pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import click
+from aioredlock import LockError
+
+from pyartcd import exectools, locks
+from pyartcd.cli import cli, pass_runtime, click_coroutine
+from pyartcd.runtime import Runtime
+
+
+async def sync_repo_to_s3_mirror(local_dir: str, s3_path: str, dry_run: bool):
```

Comment:
Why is `remove_old` parameter removed?

Reply:
In our whole codebase, the only place where `remove_old` is set to False is within `sync-for-ci` job. That translates into a `aws s3 sync --delete` invocation that can be inlined whenever we bring that job to Python. Maybe we could move this function in a dedicated `s3.py` or the like file for reuse

Reply:
`remove_old` defaults to `True` and `aws s3 sync --delete` is invoked in the current groovy code.

Reply:
Thanks. Fixed in 28bce2ef7ef7fed49b69b64e40833945ba6430ef